src/components/ConnectedFilter/index.jsx

```diff
@@ -21,25 +23,32 @@ class ConnectedFilter extends React.Component {
       initialAggsData: {},
       receivedAggsData: {},
       accessibility: ENUM_ACCESSIBILITY.ALL,
+      adminAppliedPreFilters: Object.assign({}, this.props.adminAppliedPreFilters),
+      filter: Object.assign({}, this.props.adminAppliedPreFilters),
     };
     this.filterGroupRef = React.createRef();
+    this.adminPreFiltersFrozen = JSON.stringify(this.props.adminAppliedPreFilters).slice();
```

Comment:
🤔  could I know why we want this frozen variable?

Reply:
sometimes `props` disappears so this is kind of a work around to save and recreate it?

Reply:
yeah! some piece of code was modifying my variable, but after extensive debugging, could not locate it.